kuryr_kubernetes/cni/utils.py

```diff
@@ -63,3 +66,18 @@ def __init__(self, env, cfg=None):
     def __repr__(self):
         return repr({key: value for key, value in self.__dict__.items() if
                      key.startswith('CNI_')})
+
+
+def measure_time(command):
+    """Measures CNI ADD/DEL resquest duration"""
+    def decorator(method):
+        def wrapper(obj, *args, **kwargs):
+            start_time = time.time()
+            result = method(obj, *args, **kwargs)
+            cni_request_accepted = (result[1] == httplib.ACCEPTED)
```

Comment:
It's NO_CONTENT on DEL requests. Probably better to check if it's in 200-299 range?